src/fdc_ts/core.clj

```diff
@@ -51,19 +51,23 @@
 
 ;TODO Move put to separate module
 
-(defn- auth [token ctx]
-  (= (token env) (get-in ctx [:request :headers "auth-token"])))
+(defn- auth [token project-tokens project ctx]
+  (def request-token  (get-in ctx [:request :headers "auth-token"]))
```

Comment:
I think you are definig request-token globally.
It should be a (let [request-token (get-in ...))
                           (or (= ....)